test/testfloat/testfloat.cpp

```diff
@@ -228,13 +228,11 @@ constexpr TypedValue from_bits(ValType ty, uint64_t bits) noexcept
     switch (ty)
     {
     case ValType::i32:
-        return static_cast<uint32_t>(bits);
+    case ValType::f32:  // For f32 interpreter bits as floating-point value.
```

Comment:
```suggestion
    case ValType::f32:  // For f32 interpret bits as floating-point value.
```
Should this mention that this doesn't convert to float `Value` to avoid sNan -> qNan conversion?

Reply:
Fixed.